gamemodes/attdef.pwn

```diff
@@ -450,45 +537,54 @@ public ServerOnPlayerDeath(playerid, killerid, reason)
 	}
 	else
 	{
-		switch(reason)
+		if(!RandomDeathMessages)
```

Comment:
You said people already complained about those messages being inappropriate, I think we should completely remove them then.